spec/features/user_deletes_review_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+feature "User deletes review" do
+  scenario "User deletes review" do
+
+   artwork = Art.create(
+     name: "Chinatown Community Mural",
+     location: "Surface Rd & Beach St",
+     description: "Chinese community mural commissioned
+     by the mayor's mural crew.",
+     artist: "Made up Artist",
+     category: "Graffiti"
+   )
+   user = FactoryGirl.create(:user)
+   visit root_path
+   click_link "Sign In!"
+   fill_in 'Email', with: user.email
+   fill_in 'Password', with: user.password
+
+   click_button 'Sign In'
+   visit art_path(artwork)
+
+   click_on "Add New Review"
+
+   fill_in "body", with: "This was the greatest mural I have ever seen!"
+   fill_in "rating", with: 5
+   click_on "Add"
```

Comment:
Instead of having the user create and then immediately delete the review, I think it would be more to-the-point to simply create a review in the database, and have it be associated with this particular piece of art and user. That way, when the user visits the page, there's already a review there for them to delete.